README.md

```diff
@@ -109,25 +109,25 @@ Prerequisites
 To build:
 
 ```
-make BOARD=feather_nrf52840_express all
+make BOARD=feather_nrf52840_express SD=s140 all
```

Comment:
please remove those `SD=s140`, you can add an additional build command to demonstrate using your ANT stack e.g

```
By default, the BLE stack S132/S140 is used, to build with a different softdevice stack, 

make SD=.... 
```